public/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+
+<html>
+  <head>
+    <title>iOS + Hyper</title>
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <script type="text/javascript" src="scripts.js"></script>
+
```

Comment:
They did it here too though. https://github.com/hyperoslo/hyper-says/blob/master/public/index.html